### DISCUSSION TITLE
Discover insteon address for device?

### DISCUSSION THREAD

**Discussion by f1d094** [TD22057/insteon-mqtt#527] [Q&A] [answered]
With the java interface of Universal Devices' ISY994i it is possible to use the physical pairing method to capture the address of a device when it pairs with the modem. Is there a way to do this with insteon-mqtt? I have several devices missing the address decal and expect some from EBay will be the same.

**Comment by krkeegan** [comment, marked as answer]
No, there isn't.  At least not in an easy fashion.  It is possible that with debugging turned on you may see the modem complain about a message from an unknown device.  This log entry would have the device id in it.

**Reply by f1d094** [reply]
Thank you for the quick reply!

This would be a nice feature. Being able to put insteon-mqtt into "pairing mode" and then press the button to be discovered would be very handy. I do know that the HASS insteon plugin does this, if that code is of any help. :)
